EnglishTranslation-master/QUEST_LV_0100.tsv

```diff
@@ -10660,37 +10660,45 @@ QUEST_LV_0100_20151102_010634	If it were anyone else, I would have stopped you b
 QUEST_LV_0100_20151102_010644	The ripple damage from destroying the Mind Control Tower might harm the owls.{nl}Please, fill my comrades with the energy of the forest before you destroy the Mind Control Tower.{nl}
 QUEST_LV_0100_20151102_010645	Please, gather the fragments of the Karolis Altar destroyed by the demons.{nl}Restoring the Altar will be able to recover some of the forest.{nl}
 QUEST_LV_0100_20151102_010646	I will give you the Guide Owl's Will to you.{nl}It will guide you to the place where the Karolis Altar used to be.
-QUEST_LV_0100_20151102_010647	Karolis Altar used to pump the forest energy.{nl}Ever since it got destroyed, the energy of the forest got reduced quite significantly.{nl}
-QUEST_LV_0100_20151102_010648	The Forest Altar still stands but to spread the energy throughout the entire forest, it needs Karolis Lights.{nl}The lights went out so did the spreading of the energy.{nl}
-QUEST_LV_0100_20151102_010649	If both the Karolis Altar and the Forest Sanctuary are restored. My comrades will be back to normal.{nl}We, the owls, share lives with the forest.
+QUEST_LV_0100_20151102_010647	Karolis Altar used to pump the forest energy.{nl}Ever since it got destroyed, the energy of the forest got reduced quite significantly.{nl}
+QUEST_LV_0100_20151102_010648	The Forest Altar still stands but to spread the energy {nl}throughout the entire forest, it needs Karolis Lights.{nl}The lights went out so did the spreading of the energy.{nl}
+QUEST_LV_0100_20151102_010649	If both the Karolis Altar and the Forest Sanctuary are restored. {nl}My comrades will be back to normal. We, the owls, share lives with the forest.
 QUEST_LV_0100_20151102_010650	I've been waiting for you.{nl}I knew you succeeded when I sensed the energy of the forest strengthened.
-QUEST_LV_0100_20151102_010651	I should not have held my hopes to high up... Restored Karolis Altar was not enough to revive the forest fully.{nl}The Altar of the Forest needs to be revived as well.{nl}
+QUEST_LV_0100_20151102_010651	I should not have held my hopes to high up... {nl}The restored Karolis Altar was not enough to completely revive the forest.{nl}The Altar of the Forest needs to be revived as well.{nl}
 QUEST_LV_0100_20151102_010652	Ah, the Altar of the Forest is located near the pond across there.{nl}It spreads Karolis Lights throughout the forest.{nl}
-QUEST_LV_0100_20151102_010653	The lights are almost all out..{nl}If you can re-ignite them the power of the altar will spread out.
-QUEST_LV_0100_20151102_010654	There are four Karolis Lights.{nl}Re-ignite only one light, it will go off very quickly, but re-ignite all four, the power is sustainable.{nl}
+QUEST_LV_0100_20151102_010653	The lights are almost all out...{nl}If you can re-ignite them the power of the altar will spread out.
+QUEST_LV_0100_20151102_010654	There are four Karolis Lights.{nl}Re-ignite only one light, it will go off very quickly, {nl}but re-ignite all four, the power is sustainable.{nl}

```

Comment:
Instead of the power is sustainable you can say it will sustain them all,or something to the extent where igniting all four would sustain the power so it does not go out.